app/src/main/java/io/appium/uiautomator2/model/api/gestures/PointModel.java

```diff
@@ -0,0 +1,24 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import android.graphics.Point;
+
+import io.appium.uiautomator2.model.RequiredField;
+import io.appium.uiautomator2.model.api.BaseModel;
+import io.appium.uiautomator2.model.api.ElementModel;
+
+public class PointModel extends BaseModel {
+    @RequiredField
+    public Double x;
+    @RequiredField
+    public Double y;
+
+    public PointModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)